rep-2000.rst

```diff
@@ -407,7 +406,7 @@ Dependency Requirements:
 
 " * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
 
-" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime. The versions shown for OpenEmbedded are those provided by the 3.1 Dunfell release series; the versions provided by the other supported release series are listed here: https://github.com/ros/meta-ros/wiki/Package-Version-Differences .
```

Comment:
Nitpick: new sentence should start on new line (see style guide).

Same below.